src/sentry/identity/vsts/provider.py

```diff
@@ -69,6 +78,20 @@ def get_refresh_token_params(self, refresh_token, *args, **kwargs):
             'redirect_uri': absolute_uri(oauth_redirect_url),
         }
 
+    def build_identity(self, data):
+        data = data['data']
+        user = get_user_info(data['access_token'])
+
+        return {
+            'type': 'vsts',
+            'id': user['id'],
+            'email': user['emailAddress'],
+            'email_verified': True,
```

Comment:
So, are `email` and `email_verified` used? I don't see them passed to `build_integration`

Reply:
They're used by the login pipeline which calls build_identity directly